internal/hcs/system.go

```diff
@@ -291,6 +291,10 @@ func (computeSystem *System) waitBackground() {
 	oc.SetSpanStatus(span, err)
 }
 
+func (c *System) WaitChannel() <-chan struct{} {
+	return c.waitBlock
```

Comment:
looking at the code we don't currently close `c.closeCh` anymore during `Close()`... I wonder what happened there.

Reply:
Yes! Infact I do not see closeCh being used anywhere too. Discussion on that here - https://github.com/microsoft/hcsshim/pull/1601/files#r1044855223

Reply:
We could cleanup and refactor in the later long term fix I guess